src/utils/userDataManager.ts

```diff
@@ -4,30 +4,58 @@ import * as path from 'path';
 import { HistoricalHttpRequest } from '../models/httpRequest';
 import { JsonFileUtility } from './jsonFileUtility';
 
+const restClientDir = 'rest-client';
+const rootPath = path.join(os.homedir(), `.${restClientDir}`);
+
+function getCachePath(): string {
+    if (fs.existsSync(rootPath)) {
+        return rootPath;
+    }
+
+    if (process.env.XDG_CACHE_HOME !== undefined) {
+        return path.join(rootPath, process.env.XDG_CACHE_HOME);
```

Comment:
Why do we need to join the `rootPath` and `process.env.XDG_CACHE_HOME`?

Reply:
whoops that should be os.homedir().

Reply:
actually is just needs to be process.env.X